yarn-project/cli-wallet/src/utils/wallet.ts

```diff
@@ -50,11 +49,19 @@ export class CLIWallet extends BaseWallet {
     executionPayload: ExecutionPayload,
     from: AztecAddress,
     userFee?: UserFeeOptions,
-  ): Promise<TxExecutionRequest> {
+  ): Promise<{ txRequest: TxExecutionRequest; gasEstimationsSimulationResults?: TxSimulationResult }> {
```

Comment:
This function name is now broken because it sneaks in the return value the gas estimation tingy. This is not clear from the function name.